extensions/xcode/src/components/xcode-simulator-list/xcode-simulator-list.component.tsx

```diff
@@ -1,12 +1,25 @@
 import { useCachedPromise } from "@raycast/utils";
 import { XcodeSimulatorService } from "../../services/xcode-simulator.service";
+import { XcodeSimulatorStateDropdown } from "./xcode-simulator-list-dropdown.component";
 import { List } from "@raycast/api";
 import { XcodeSimulatorListItem } from "./xcode-simulator-list-item.component";
+import { useState } from "react";
+import { XcodeSimulatorStateFilter } from "../../models/xcode-simulator/xcode-simulator-state-filter.model";
 
 export function XcodeSimulatorList(): JSX.Element {
-  const xcodeSimulatorGroups = useCachedPromise(XcodeSimulatorService.xcodeSimulatorGroups);
+  const [simulatorStateFilter, setSimulatorStateFilter] = useState<XcodeSimulatorStateFilter>(
+    XcodeSimulatorStateFilter.all
+  );
+  const xcodeSimulatorGroups = useCachedPromise(XcodeSimulatorService.xcodeSimulatorGroups, [simulatorStateFilter]);
+
+  const onSimulatorStateFitlterChange = (newValue: XcodeSimulatorStateFilter) => {
```

Comment:
Typo `onSimulatorStateFitlterChange` should be changed to `onSimulatorStateFilterChange`

Reply:
✅ Fixed